openslides_backend/action/actions/user/set_password_self.py

```diff
@@ -44,12 +44,15 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
             raise ActionException("Can't set password for anonymous")
 
         instance["id"] = self.user_id
-        if is_temporary(self.datastore, instance):
+        if is_meeting_scope(self.datastore, instance["id"]):
```

Comment:
I think you should use the get_user_scope method from user_scope_permission_check_mixin. It will give you the scope and, if it is meeting, the meeting_id.

Reply:
Ah yes, I did not see that scopes were also important for these actions (I checked only the changes in `update_self`...). Please rebase on #717 and use the new functions there.

Reply:
Rebase to #717 and use the get scope function.